src/context/account.context.tsx

```diff
@@ -74,19 +52,14 @@ export const AccountProvider = ({ children }: AccountContextProps) => {
     () =>
       new ENS(
         currentNetwork.label === "Sepolia"
-          ? { ...currentNetwork.config, rpcUrl: "http://rpc.sepolia.org" }
+          ? { ...currentNetwork.config, rpcUrl: "https://sepolia.dev.fairdatasociety.org/" }
```

Comment:
This check now can be completely removed. It was used for gas estimation using the previous method.